src/components/button/__tests__/Button.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Button from '../Button';
+
+describe('Component - Button', () => {
+  test('it renders', async () => {
```

Comment:
pp use `it` instead of `test` 
```suggestion
  it('renders', async () => {
```